### ISSUE TITLE
User Read—Incorrent Content-Type

### ISSUE THREAD

**Issue by alunduil** [alunduil/margarine#18]
Reading a user returns the incorrect Content-Type:

```
======================================================================
FAIL: User Read—Submitted,Complete
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_margarine/test_unit/test_blend/test_user.py", line 129, in test_user_read_submitted_complete
    self.assertEqual('application/json', response.headers.get('Content-Type'))
AssertionError: 'application/json' != 'text/html; charset=utf-8'
-------------------- >> begin captured logging << --------------------
margarine.blend.user: DEBUG: user: {'username': 'alunduil', '_id': ObjectId('5201af224ee7ca5cbf3adf43'), 'email': 'alunduil@alunduil.com'}
--------------------- >> end captured logging << ---------------------
```

**Comment by alunduil** [comment]
```
======================================================================
FAIL: Blend::User Read—Submitted,Complete
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_margarine/test_unit/test_blend/test_user.py", line 129, in test_user_read_submitted_complete
    self.assertEqual('http://margarine.raxsavvy.com', response.headers.get('Access-Control-Allow-Origin'))
AssertionError: 'http://margarine.raxsavvy.com' != None
-------------------- >> begin captured logging << --------------------
test_margarine/test_unit/__init__.py:92: 32440: mocks: set(['keyspace', 'container', 'collection', 'channel'])
test_margarine/test_unit/__init__.py:93: 32440: mock_mask: set(['container'])
test_margarine/test_unit/__init__.py:94: 32440: to_mock: set(['keyspace', 'channel', 'collection'])
/home/alunduil/Projects/savvy/margarine/margarine/blend/user.py:282: 32440: user: {'username': 'alunduil', '_id': ObjectId('522673bb4ee7ca7eb8476bc9'), 'email': 'alunduil@alunduil.com'}
--------------------- >> end captured logging << ---------------------
```

**Comment by alunduil** [comment]
```
======================================================================
FAIL: Blend::User Read—Submitted,Complete
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_margarine/test_unit/test_blend/test_user.py", line 129, in test_user_read_submitted_complete
    self.assertEqual('http://margarine.raxsavvy.com', response.headers.get('Access-Control-Allow-Origin'))
AssertionError: 'http://margarine.raxsavvy.com' != 'alunduil.com'
-------------------- >> begin captured logging << --------------------
test_margarine/test_unit/__init__.py:92: 3705: mocks: set(['keyspace', 'container', 'collection', 'channel'])
test_margarine/test_unit/__init__.py:93: 3705: mock_mask: set(['container'])
test_margarine/test_unit/__init__.py:94: 3705: to_mock: set(['keyspace', 'channel', 'collection'])
/home/alunduil/Projects/savvy/margarine/margarine/blend/user.py:282: 3705: user: {'username': 'alunduil', '_id': ObjectId('522677044ee7ca0e79ab6445'), 'email': 'alunduil@alunduil.com'}
/home/alunduil/Projects/savvy/margarine/margarine/parameters.py:137: 3705: len(parameters): 0
/home/alunduil/Projects/savvy/margarine/margarine/parameters.py:138: 3705: parsed? False
/home/alunduil/Projects/savvy/margarine/margarine/parameters.py:288: 3705: Adding parameters to default.
/home/alunduil/Projects/savvy/margarine/margarine/parameters.py:289: 3705: parameters: ()
/home/alunduil/Projects/savvy/margarine/margarine/parameters.py:227: 3705: Parsing Configuration Files
/home/alunduil/Projects/savvy/margarine/margarine/parameters.py:337: 3705: Parameters not parsed.
/home/alunduil/Projects/savvy/margarine/margarine/parameters.py:339: 3705: Searching for key: server.domain
/home/alunduil/Projects/savvy/margarine/margarine/parameters.py:347: 3705: default: alunduil.com
/home/alunduil/Projects/savvy/margarine/margarine/parameters.py:349: 3705: Checking Environment
/home/alunduil/Projects/savvy/margarine/margarine/parameters.py:357: 3705: environ_key: __MAIN__.PY_SERVER_DOMAIN
/home/alunduil/Projects/savvy/margarine/margarine/parameters.py:361: 3705: value: alunduil.com
/home/alunduil/Projects/savvy/margarine/margarine/parameters.py:363: 3705: Checking Configuration File
/home/alunduil/Projects/savvy/margarine/margarine/parameters.py:365: 3705: self._configuration_files: {'/etc/margarine/margarine.ini': <ConfigParser.SafeConfigParser instance at 0x321ffc8>}
/home/alunduil/Projects/savvy/margarine/margarine/parameters.py:372: 3705: type(configuration_file): <type 'instance'>
/home/alunduil/Projects/savvy/margarine/margarine/parameters.py:373: 3705: configuration_file: <ConfigParser.SafeConfigParser instance at 0x321ffc8>
/home/alunduil/Projects/savvy/margarine/margarine/parameters.py:382: 3705: value: alunduil.com
/home/alunduil/Projects/savvy/margarine/margarine/parameters.py:384: 3705: Checking CLI Arguments
/home/alunduil/Projects/savvy/margarine/margarine/parameters.py:388: 3705: self.arguments.server_domain: alunduil.com
/home/alunduil/Projects/savvy/margarine/margarine/parameters.py:393: 3705: argument_value: alunduil.com
/home/alunduil/Projects/savvy/margarine/margarine/parameters.py:398: 3705: value: alunduil.com
--------------------- >> end captured logging << ---------------------
```

**Comment by alunduil** [comment]
After a second look.  This is now in the domain of the default value of Allow-Control-Allow-Origin, issue #15.  As a result the changes contained in this branch should be reviewed for merge potential.

**Comment by hhoover** [comment]
closing in favor of issue #15

**Comment by alunduil** [comment]
Merged in commits c909a86..40cec11.
